runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBoxOverlay.java

```diff
@@ -76,13 +86,18 @@
 		this.config = config;
 		this.skill = skill;
 		this.icon = icon;
+		this.xpPanel = xpPanel;
 		panel.setBorder(new Rectangle(BORDER_SIZE, BORDER_SIZE, BORDER_SIZE, BORDER_SIZE));
 		panel.setGap(new Point(0, XP_AND_PROGRESS_BAR_GAP));
 		panel.setPreferredSize(new Dimension(PANEL_PREFERRED_WIDTH, 0));
 		iconXpSplitPanel.setBorder(XP_AND_ICON_COMPONENT_BORDER);
 		iconXpSplitPanel.setBackgroundColor(null);
 		iconXpSplitPanel.setPreferredSize(new Dimension(PANEL_PREFERRED_WIDTH, 0));
 		getMenuEntries().add(new OverlayMenuEntry(RUNELITE_OVERLAY_CONFIG, OPTION_CONFIGURE, "XP Tracker overlay"));
+		getMenuEntries().add(new OverlayMenuEntry(RUNELITE_OVERLAY, OPTION_REMOVE, skill.getName()));
+		getMenuEntries().add(new OverlayMenuEntry(RUNELITE_OVERLAY, OPTION_RESET, skill.getName()));
+		//getMenuEntries().add(new OverlayMenuEntry(RUNELITE_OVERLAY, OPTION_UNPAUSE, skill.getName()));
```

Comment:
Why is this line and other lines relating to pausing commented?

Reply:
At first I was trying to have the menu option switch between pause and unpause based on the state.

Reply:
Is that not going to be added?

Reply:
I attempted it. Pausing from the panel would require access to the overlay to change menu entries. I switched it to "Toggle tracking" for pause and unpause actions.

Reply:
You can just extend OverlayMenuEntry

Reply:
For what class? All the relevant classes extends to something already.